test/helpers/exec.js

```diff
@@ -25,15 +26,15 @@ const compareStatObjects = (a, b) => {
 	return 1;
 };
 
-exports.fixture = async (...args) => {
+exports.fixture = async (args, options = {}) => {
```

Comment:
At the existing tests for [node-assertions](https://github.com/avajs/ava/blob/master/test-tap/integration/node-assertions.js) custom environment variables where needed. I updated this code to make it simpler to provide overrides for them. A consequence of this change, is that *args* should be provided as an array now (just like it happens in *execa*) and I had to update the existing tests to also specify args that way.

Please let me know if I should update this to use rest args.

Reply:
Yea… I guess this was inevitable 😄